kenshin-profile-app/kenshin-profile-app/ContentView.swift

```diff
@@ -9,13 +9,96 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
+        ZStack(alignment: .top) {
+
+            Image("background2")
+                .resizable()
+                .frame(width: 435, height: 860)
+        
+            MapView()
+                .frame(height: 220)
+                .blur(radius: 1)
+            
+            VStack {
+
+                Spacer()
+                
+                HStack {
+                    
+                    Text("    📍Chiba,Japan")
```

Comment:
文字列の前に余白が入ってるから余白は消そう。
```suggestion
                    Text("📍Chiba,Japan")
```